apps/motionscope/analyze.py

```diff
@@ -128,7 +128,12 @@ def func(val):
     def export_gs(self, filename):
         data = self.data_frame()
         gtc = self.gcloud.get_interface("KtabularClient")
-        sheet = gtc.create(filename,data)
+        sheet = gtc.lookup(filename)
+        if (sheet != None):
```

Comment:
A couple of notes:
In python, no parenthesis needed around if clauses.  And:
if sheet is not None 
is preferred over
if sheet!=None
Or you can do 
if not sheet
It's odd (agreed).  But there is only one None object and comparing None to stuff gets some people excited, even though it works.  :)  
http://jaredgrubb.blogspot.com/2009/04/python-is-none-vs-none.html